src/webview/js/js.js

```diff
@@ -908,6 +913,20 @@ documentBody.addEventListener('click', (e: MouseEvent) => {
   }
 });
 
+const revealMutedMessages = (target: Element) => {
```

Comment:
This should get a bit of jsdoc to describe what its behavior is supposed to be. It looks like it's fairly subtle.

Reply:
Done. Let me know if this needs any more elaboration.